test/functional/admin/users_controller_test.rb

```diff
@@ -70,4 +70,14 @@ class Admin::UsersControllerTest < ActionController::TestCase
     put :update, params: { id: @user.id, user: { world_location_ids: [world_location.id] } }
     assert_response :forbidden
   end
+
+  view_test "show: hides Edit link if user has no gds editor permission to edit the record or display the link if user has gds editor permission." do
```

Comment:
I think this might be clearer as two separate tests—one with permission, and one without?